resources/faq.md

```diff
@@ -51,25 +51,6 @@ In this documentation, we provide this <a id="downloadMovie" href="/movies.json"
 
 More datasets and setting configurations are available [in this repository](https://github.com/meilisearch/datasets/).
 
-## I did a call to an API route, and I only got an `updateId` as a response. What does it mean?
```

Comment:
We should maybe replace this FAQ answer w/ something

Reply:
I wasn’t sure if we needed something here so I removed it? Maybe explain the summarized task response the users get?